mne/gui/_coreg_gui.py

```diff
@@ -859,13 +1222,20 @@ class CoregPanel(HasPrivateTraits):
     scale_labels = DelegatesTo('model')
     copy_annot = DelegatesTo('model')
     prepare_bem_model = DelegatesTo('model')
-    save = Button(label="Save As...")
```

Comment:
I had "Save As" based on the UI convention that "Save" usually overwrites a previously opened file, whereas "Save As..." shows a dialog. For example Safari has "Save As..." but not "Save" in its File menu. I think the Button is not the widest item currently and should still fit :)